source/connect-to-govwifi.html.erb

```diff
@@ -4,7 +4,28 @@ title: Connect to GovWifi
 
 <div class="container">
   <div class="grid-row">
-    <h1 class="heading-large">Connect staff to GovWifi</h1>
+    <div class="column-one-third">
+      <div>
+        <%= partial 'shared/support/sidebar' %>
+      </div>
+    </div>
+    <div class="column-two-thirds">
+      <h1 class="heading-large">Connect to GovWifi</h1>
+      <p>If you have a government or public sector organisation email address, you need to sign up to get to get your unique username and password.</p>
+      <p>You can check if your government or public sector organisation email address is eligible for GovWifi.</p>
+      <div>
+        <p>You will need to use your government or public sector organisation email address to:</p>
+        <ul class="list list-bullet">
+          <li>send a blank email to signup@wifi.service.gov.uk</li>
+          <li>leave the subject line blank</li>
+          <li>read and accept the terms and conditions, and our privacy notice.</li>
+        </ul>
+      </div>
+      <p>We will email your username and password in reply.</p>
+      <p>Once you’re set up, you can use the same username and password on all the devices you connect to GovWifi.</p>
+      <div class="govuk-inset-text">
+        <a href="/connect-visitors-to-govwifi">Connect to GovWifi without a government or public sector email address</a>
```

Comment:
Probably want to use the ERB links here eg
<%= link_to "something", "/google" %>